mozilla_aws_cli/cli.py

```diff
@@ -200,6 +200,9 @@ def main(batch, config, cache, output, print_url,
     if web_console and print_url:
         raise click.exceptions.UsageError(
             "Cannot print URL to output and redirect to web console")
+    if output in ("awscli", "shared") and profile is None:
+        raise click.exceptions.UsageError(
+            "You must specify a profile name with `awscli` or `shared` output")
```

Comment:
Is this true? I'm able to use the the tool today with those outputs without asserting a profile name

```
gene@judy ~/code/github.com/gene1wood $(maws -o awscli)
Environment variables set for role arn:aws:iam::371522382791:role/MAWS-Admin
gene@judy ~/code/github.com/gene1wood (infosec-prod-MAWS-Admin)$ 
```

Reply:
I was having issues with it, but I think I figured out why and fixed it.